website/docs/cloud-docs/api-docs/run-tasks/run-task-stages-and-results.mdx

```diff
@@ -40,7 +40,7 @@ The run task stage status is found in `data.attributes.status`, and you can refe
 | `running`           | The run task stage is executing one or more tasks, which have not yet completed.                                    |
 | `passed`            | All of the run task results in the stage passed.                                                                    |
 | `failed`            | One more results in the run task stage failed.                                                                      |
-| `awaiting_override` | Indicates that the task stage is waiting for user input, once overridden it re-enters back into the `running` state |
+| `awaiting_override` | The task stage is waiting for user input. Once overridden, the run returns to the `running` state.                  |
```

Comment:
Also just checking - there's nothing special about these for OPA specifically is there? Or...I just want to make sure that these all apply to both

```suggestion
| `awaiting_override` | The task stage is waiting for user input. Once a user manually overrides the failed policy checks, the run returns to the `running` state.                  |
```

Reply:
@glennsarti Correct me if I am wrong, but. I think this is applicable only to OPA as of now, but will apply to both OPA and sentinel in the future, when sentinel is also part of the task stage?

Reply:
Whoops didn't see this ping. So the verbiage we're using at the moment is "It's a Policy Override". We've tried to avoid "Policy Checks" because that's the language for current Sentinel.  While the old text is _technically_ correct, I think Laura's text is more user facing.